src/static/store/modules/app/types.ts

```diff
@@ -11,4 +11,6 @@ export interface AppStore {
   url: string;
   screenViewMode: '3-up' | 'onlyDiff' | 'loupe' | 'swipe' | 'onionSkin';
   testsViewMode: 'collapseAll' | 'expandAll' | 'expandErrors' | 'expandRetries';
+  running: boolean;
+  config?: any;
```

Comment:
Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.